ubuntu-xenial-16.04-cloudimg-console.log

```diff
@@ -0,0 +1,626 @@
+[    00000 Iniializing grp subss ust

```

Comment:
I assume this is something to do with our upgraded Vagrant setup 🤔Could you delete it or make sure it's covered by the .gitignore?

Reply:
oh good spot, will do.